ts/src/renderer/phaser/classes/PhaserUnit.ts

```diff
@@ -88,6 +90,21 @@ class PhaserUnit extends Phaser.GameObjects.Container {
 				label.visible = false;
 			});
 
+		this.fadingTextListener =
+			unit.on('fading-text', (config: {
```

Comment:
could we encapsulate this in a function and add to evtListeners the same way we do in the other classes?

Reply:
also why do we call the object parameter `config` when we have named the same sort of thing `data` everywhere else?

Reply:
also inside fadingTextListener we already have 'data' too 
![image](https://user-images.githubusercontent.com/62058278/177167247-70d1000d-37d0-4eb1-8491-75e1f7f339e8.png)